parts/k8s/windowsnodecleanup.ps1

```diff
@@ -0,0 +1,116 @@
+<#
+.DESCRIPTION
+    This script is intended to be run each time a windows nodes is restarted and performs
+    cleanup actions to help ensure the node comes up cleanly.
+#>
+
+$global:LogPath = "c:\k\windowsnodecleanup.log"
+$global:HNSModule = "c:\k\hns.psm1"
+
+$global:MasterSubnet = "{{MasterSubnet}}"
```

Comment:
nit: can you add a comment that these lines are replaced in kuberneteswindowssetup.ps1, not a golang template? Someone might assume this is a template when they see `{{ ... }}`

Reply:
will do